### PR TITLE
admission api: cut off api from k8s.io/apiserver

### PR DESCRIPTION
**Pull request by sttts**
This caused an import cycle: api -> apiserver -> api

Fixes https://github.com/kubernetes/kubernetes/issues/51212

### CODE REVIEW

File: staging/src/k8s.io/api/admission/v1alpha1/types.go [hunk 1]
```diff
@@ -48,7 +47,7 @@ type AdmissionReviewSpec struct {
 	// +optional
 	OldObject runtime.RawExtension `json:"oldObject,omitempty" protobuf:"bytes,3,opt,name=oldObject"`
 	// Operation is the operation being performed
-	Operation admission.Operation `json:"operation,omitempty" protobuf:"bytes,4,opt,name=operation"`
+	Operation Operation `json:"operation,omitempty" protobuf:"bytes,4,opt,name=operation"`
```
Comment (deads2k) [inline staging/src/k8s.io/api/admission/v1alpha1/types.go]: @liggitt is this going to mess with proto encoding?  I think they were both strings in the end.

**Comment by caesarxuchao** [comment]
LGTM.

@stevekuznetsov could you help investigate why the importverifier doesn't catch the violation? Thanks.

**Comment by stevekuznetsov** [comment]
@caesarxuchao I've identified the bug and supplied PR #51216 to fix it

File: staging/src/k8s.io/api/admission/v1alpha1/types.go (hunk 1 above)
Comment (liggitt) [inline staging/src/k8s.io/api/admission/v1alpha1/types.go]: Yeah, the marshal method didn't change, still just a copy of []byte

**Comment by deads2k** [comment]
Fixes a cycle without changing external API.  approving

**Pull request merged**